script/parse-examples

```diff
@@ -17,7 +17,7 @@ function clone_repo {
   fi
 
   pushd "$path" > /dev/null
-  if [ "$(git rev-parse head)" != "$sha"  ]; then
+  if [ "$(git rev-parse HEAD)" != "$sha"  ]; then
```

Comment:
Ha, I'm just realizing today that this only worked because the main filesystems on macos and windows are case insensitive.